adcircpy/forcing/winds/best_track.py

```diff
@@ -119,22 +119,20 @@ def summary(self, output: Union[str, PathLike] = None):
         track_length = self.track_length
         duration = self.duration
         min_central_pressure = np.min(self.central_pressure)
-        max_wind_speed = np.max(self.df["max_sustained_wind_speed"])
-        start_loc = (self.df["longitude"][0], self.df["latitude"][0])
-        end_loc = (self.df["longitude"].iloc[-1], self.df["latitude"].iloc[-1])
+        max_wind_speed = np.max(self.df['max_sustained_wind_speed'])
+        start_loc = (self.df['longitude'][0], self.df['latitude'][0])
+        end_loc = (self.df['longitude'].iloc[-1], self.df['latitude'].iloc[-1])
         if output is None:
-            _storm = f"Summary of storm: {self.storm_id}"
+            _storm = f'Summary of storm: {self.storm_id}'
```

Comment:
Just so that we keep it consistent (and efficient), strings should be built by appending to a list, and then using the string join method, for example:
```python
f = ['this', 'is', 'my', 'string']
print(' '.join([*f, 'space', 'delimited']))
print('\n'.join([*f, 'newline', 'delimited']))
```

Reply:
okay done.